rust/Cargo.toml

```diff
@@ -26,6 +26,7 @@ regex = "1"
 chrono = "0"
 uuid = { version = "0.8", features = ["serde", "v4"] }
 lazy_static = "1"
+itertools = "0"
```

Comment:
Is this used anywhere?

Reply:
It is used for `into_group_map` op. I think your implementation is better and I will remove this later.